os/wqueue/work_process.c

```diff
@@ -134,11 +134,12 @@ void work_process(FAR struct wqueue_s *wqueue, int wndx)
 	clock_t ctick;
 	clock_t next;
 
-#ifdef CONFIG_SCHED_WORKQUEUE_SORTING
+#if (defined(CONFIG_SCHED_LPWORK) && CONFIG_SCHED_LPNTHREADS > 0) || defined(CONFIG_SCHED_WORKQUEUE_SORTING)
 	sigset_t set;
 	sigemptyset(&set);
 	sigaddset(&set, SIGWORK);
```

Comment:
Regardless of this PR, is this necessary all the time? Isn't it enough setting once at start of work queue?